tests/integration/components/cf-field-test.js

```diff
@@ -30,9 +30,15 @@ module("Integration | Component | cf-field", function(hooks) {
           label: "Test2",
           isRequired: "true",
           isHidden: "true",
-          meta: {
-            formatValidators: ["email"]
-          },
+          meta: { formatValidators: ["email"] },
+          __typename: "TextQuestion"
+        },
+        {
+          slug: "question-3",
+          label: "Test3",
+          isRequired: "true",
+          isHidden: "true",
```

Comment:
Nitpick: Shouldn't this be `false`? (same above)

Reply:
Yes you are right. I didn't recognize it.